### DISCUSSION TITLE
If notes are too long the scrollbar disappears

### DISCUSSION THREAD

**Discussion by mxng** [standardnotes/forum#2523] [General] [answered]
The longer the notes, the smaller the length of the scrollbar in the note. There's a few times the scrollbar became so small it is not rendered (maybe due to screen dimensions?) and therefore became invisible. Maybe it's possible to set a minimum of 1px or something for the scrollbar length? 

Not that important as I doubt many people make big notes, and I can separate the notes. Just thought I should ask in case.

**Comment by atmoio** [comment]
Any chance you can share a screenshot of what you mean?

**Comment by mxng** [comment]
https://imgur.com/a/eJHdKuR

In the screenshots, the first one is the one with a note with 54k words. The scrollbar area shows a blank. However, I can still drag an invisible scrollbar if I click at the correct spot where the scrollbar should be.
In the second screenshot, a note with 34k words, the scrollbar is still visible.
In the third screenshot, a note with 40k words shows the scrollbar getting thinner.

I think at some point the scrollbar becomes so small that it is invisible but still clickable. If I hold my click so that the scrollbar glides under my cursor (like how you click on the scrollbar area, but not at the scrollbar itself), and then click and drag normally, I am still able to scroll with the invisible scrollbar.

**Comment by atmoio** [comment]
Ah I see. I've seen something similar before. Will have to investigate. But 54k words in one note?? How is performance with that?

**Comment by mxng** [comment]
Thanks!

Not much problem with that size at all, surprisingly :) It only lag or hang when I switch editor type. There was once after switching the notes all showed blank and I panicked, but within a few moments the notes were displaying normally again. Scrolling and searching are still as smooth as butter.

**Comment by atmoio** [comment]
Well, glad to hear that!

**Comment by myreli** [comment, marked as answer]
It seems like the recent client versions no longer have this problem:

<img width="803" alt="image" src="https://user-images.githubusercontent.com/17554234/218591940-70fba9cd-105d-4b6a-8394-6c08c404e6c5.png">
